### ISSUE TITLE
The PDF Preview Pane doesn't recognize picture captions as a clickable style

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#1011]
The PDF Preview Pane doesn't recognize picture captions as a clickable style. If I try to right-click on a picture caption, I simply get the preview's zoom options.

Expected results:
We can right-click on the captions to bring up the Styles dialog for that style too. 

(Every time I try to find the captions style in the Styles list, I have a very hard time finding it, so I was hoping this would make it easier. Alas, not yet anyway.)

Well, cool, I just noticed the blue \fig listed next to the Caption: dialog that jumps me to the correct style! So, there is a way to quickly find it. Just not through the PDF Preview pane yet.

**Comment by markpenny** [comment]
Yes, glad you found the clickable link to \fig style on the Details tab of the Pictures page (or pane).

Hopefully next year (after further development of the right-click context menu in the PDF Preview Pane) we will be able to do some more things with illustrations. Here's a mockup of what that might look like:
![Image](https://github.com/user-attachments/assets/262d4169-3271-48f9-9a42-a58f5e0c25b1)
Let us know what else you'd like to do when you click on an illustration!

**Comment by davidc86** [comment]
Thanks for the reminder on the mockup. I saw that in the Issues list, great ideas! I gave it a Thumbs Up.

________________________________

**Comment by markpenny** [comment]
The latest version enables you to: 

(a) Show Details... (which takes you to the Pictures tab, and selects the appropriate picture on the Details sub-tab, where you can edit the caption

(b) change the style of \fig (jumps to the \fig style in the Styles editor)

(c) Send Ref to Paratext (which enables you to fix the caption in Paratext, which is best practice). We still need to update the code to make use of updated captions in PT without clobbering the remaining details in the PicList, but that will come soon.

![Image](https://github.com/user-attachments/assets/2d074b6e-f3c5-41ba-805f-5e3e8163fe3d)

**Comment by davidc86** [comment]
This is sweet! Thank you. I just used in in Matthew.
